nikola/utils.py

```diff
@@ -1272,15 +1291,49 @@ def publish_extensions(self, handler):
             })
             handler.endElement("atom:link")
 
+        for tag in ('author', 'summary'):
+            _simple_itunes_tag(self, handler, tag)
+        _itunes_image_tag(self, handler)
+        _itunes_explicit_tag(self, handler)
+
+        if self.itunes_name or self.itunes_email:
+            handler.startElement("itunes:owner", {})
+            _simple_itunes_tag(self, handler, 'name')
+            _simple_itunes_tag(self, handler, 'email')
+            handler.endElement("itunes:owner")
+
+        if self.itunes_categories:
+            for category_entry in self.itunes_categories:
+                try:
+                    category, subcategories = category_entry
+                except ValueError:
+                    category = category_entry[0]
+                    subcategories = ()
+                handler.startElement("itunes:category", {'text': category})
+                for subcategory in subcategories:
+                    handler.startElement("itunes:category", {'text': subcategory})
+                    handler.endElement("itunes:category")
+                handler.endElement("itunes:category")
+
 
 class ExtendedItem(rss.RSSItem):
     """Extended RSS item."""
 
-    def __init__(self, **kw):
+    def __init__(self, itunes_author=None, itunes_subtitle=None,
```

Comment:
Doesn't that kind of reading-into-kwargs defeat static code analysis? If you don't declare what arguments you take in your function definition, you're unnecessarily hiding information.

At least, that's been my take on it in the past.